src/vm/codeman.h

```diff
@@ -433,11 +433,11 @@ typedef struct _HeapList
     DWORD               cBlocks;        // Number of allocations
     bool                bFull;          // Heap is considered full do not use for new allocations
     bool                bFullForJumpStubs; // Heap is considered full do not use for new allocations of jump stubs
-
-#if defined(_TARGET_AMD64_)
+#if defined(_TARGET_ARM64_)
+    BYTE                padding[2];
```

Comment:
It would make sense to add a static assert verifying that the CLRPersonalityRoutine is properly aligned (I guess that's the point of the padding here)